setup.py

```diff
@@ -1,20 +1,21 @@
 import setuptools
 
-with open('README.md') as f:
+with open("README.md") as f:
     long_description = f.read()
 
 setuptools.setup(
-      name='py-repo-root',
-      version='1.1.2',
-      license='MIT',
-      description='Python utility for cleaner handling of paths',
-      long_description=long_description,
-      long_description_content_type='text/markdown',
-      url='https://github.com/pashminacameron/py-repo-root',
-      packages=setuptools.find_packages(),
-      python_requires='>=3.7.0',
-      include_package_data=True,
-      author='Pashmina Cameron, Henry Jackson Flux',
-      author_email='pashabhi@yahoo.com',
-      install_requires=[],
-      zip_safe=False)
+    name="py-repo-root",
+    version="1.2.0",

```

Comment:
Maybe this should be `2.0.0` given this change will break backward compatibility. It may also make sense to hold off upgrading major version until all the planned changes are in, thus `2.0.0` will get bundled in with the last change to go in (i.e. git submodules)?

Reply:
I would maybe go for the bump now, as you say it is the first breaking change, and we don't foresee that later changes will be breaking.